asv_bench/benchmarks/pandas/indexing.py

```diff
@@ -0,0 +1,406 @@
+"""
```

Comment:
we definitely need to reduce this - I think we have something tested already and we don't need such large number fof cases, let's decide what we really want to leave - eg multiindexing, what else is valuable?